content/jmccann/hipchat/index.md

```diff
@@ -50,12 +50,32 @@ The Hipchat plugin supports reading credentials from the Drone secret store. Thi
 ```diff
 pipeline:
   slack:
-    image: plugins/hipchat
+    image: jmccann/drone-hipchat
     room: my-room
 -   auth_token: my-auth-token
 ```
 
-The above `auth_token` Yaml attribute can be replaced with the `HIPCHAT_AUTH_TOKEN` secret environment variable. Please see the Drone documentation to learn more about secrets.
+The above `auth_token` Yaml attribute can be replaced with the `HIPCHAT_AUTH_TOKEN` secret environment variable.
+
+Use the command line utility to add secrets to the store:
+
+```nohighlight
+drone secret add --image=jmccann/drone-hipchat \
+    octocat/hello-world HIPCHAT_AUTH_TOKEN abcd1234
+```
+
+Don't forget to sign the Yaml after making changes:
+
+```nohighlight
+drone sign octocat/hello-world
+```
+
+Please see the [Drone documentation](http://readme.drone.io/0.5/secrets-with-plugins/) to learn more about secrets.
+
```

Comment:
I've starting removing the command line examples from the docs, since people were literally copy/pasing the commands with `octocat/hello-world` and wondering why it didn't work